test/spec/hooks/useElementVisible.spec.js

```diff
@@ -0,0 +1,118 @@
+import { renderHook } from '@testing-library/preact-hooks';
+
+import { waitFor } from '@testing-library/preact';
+
+import { useElementVisible } from 'src/hooks';
+
+import TestContainer from 'mocha-test-container-support';
+
+
+describe('hooks/useElementVisible', function() {
+
+  let container;
+  beforeEach(function() {
+    container = TestContainer.get(this);
+  });
+
+
+  it('should return true for visible element', async function() {
+
+    // given
+    const element = global.document.createElement('div');
+    container.appendChild(element);
+
+    // when
+    const { rerender, result } = renderHook(() => useElementVisible(element));
+
+    // then
+    await waitFor(() => {
+      rerender();
+      expect(result.current).to.be.true;
+    });
```

Comment:
[not blocking] Since this and the following two cases test a static result, we could even check the immediate result of the hook. This way we can ensure that the value is correct from the beginning, and not changed in the meantime (eventually correct).

Reply:
So
```suggestion
expect(result.current).to.be.true
```

(async can be removed as well)